code/modules/projectiles/ammunition/ballistic/rifle.dm

```diff
@@ -26,3 +26,16 @@
 	caliber = "40mm"
 	icon_state = "40mmHE"
 	projectile_type = /obj/item/projectile/bullet/a40mm
+
+// 5.7*28mm (M-90gl)

```

Comment:
Why does the M90GL (the F2000 analogue) not fire 5.56 and instead fire a high-power pistol round?

Reply:
>assault rifle
>Underbarrel grenade launcher
>P90 analogue

Reply:
It's a game after all, it has its artistic license moments.